pubg_python/domain/telemetry/objects.py

```diff
@@ -100,7 +100,7 @@ def from_dict(self):
         self.vehicle_type = self._data.get('vehicleType')
         self.vehicle_id = self._data.get('vehicleId')
         self.health_percent = self._data.get('healthPercent')
-        self.fuel_percent = self._data.get('fuelPercent')
+        self.fuel_percent = self._data.get('feulPercent')
```

Comment:
Can we add a `# TODO: update when fixed in API` above this line? Just so we don't think it's our typo in the future.